src/main/java/com/ivanmostovyi/demo/service/MessageService.java

```diff
@@ -1,19 +1,23 @@
 package com.ivanmostovyi.demo.service;
 
-import com.ivanmostovyi.demo.domain.User;
-import com.ivanmostovyi.demo.dto.MessageDto;
+import com.ivanmostovyi.demo.dto.InboxMessageDto;
 import com.ivanmostovyi.demo.dto.MessageFormDto;
+import com.ivanmostovyi.demo.dto.OutboxMessageDto;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 
 public interface MessageService {
 
-    void create(MessageFormDto messageFormDto, User senderUser);
+    Long GMAIL_SUPPORT_ID = 1L;
```

Comment:
Add migration to create this user.
Disable login for this user.

Reply:
Also, use username here, not id, because we don't want to specify id in the migration